tests/session/cmd/test_sequential.py

```diff
@@ -45,6 +45,7 @@ def _cmd(value: int) -> str:
 
 
 @pytest.mark.integration()
+@pytest.mark.xfail(reason="#2720")
```

Comment:
I've already rejected this, please remove.

Reply:
It's the same commit. I'm currently developing on top of that other PR because without this marker, the tests fail on main and it's difficult to troubleshoot the problem when there are tests failing for other reasons. I'll plan to rebase off of this workaround commit once I have a fix worked out.